azure-pipelines.yml

```diff
@@ -18,7 +18,7 @@ jobs:
   steps:
   - task: UseDotNet@2
     inputs:
-       version: '3.0.100'
+       version: '3.1.100'
```

Comment:
@Mpdreamz Any reason docs can be set to 3.1.202 as well here?

Reply:
Updated thanks for catching that :)